src/zui/ZUIToggleButton/index.tsx

```diff
@@ -1,66 +1,121 @@
-import React from 'react';
-import { ToggleButton, ToggleButtonGroup } from '@mui/material';
+import { SvgIconTypeMap, ToggleButton, ToggleButtonGroup } from '@mui/material';
+import { OverridableComponent } from '@mui/material/OverridableComponent';
+import { makeStyles } from '@mui/styles';
+import { FC } from 'react';
 
-interface Buttons {
-  value: string;
+const useStyles = makeStyles((theme) => ({
+  toggleButtonGroup: {
+    '& .MuiToggleButton-root': {
+      borderColor: theme.palette.primary.light,
+      color: theme.palette.primary.main,
+    },
+    '& .MuiToggleButton-root.Mui-selected': {
+      backgroundColor: theme.palette.primary.main,
+      color: theme.palette.primary.contrastText,
```

Comment:
I believe this needs to go back to black on `:hover`, right?

Reply:
dammit you're right. There were no design specs for the focus/hover/etc states so i forgot about that...